trunk-recorder/call.cc

```diff
@@ -179,6 +183,29 @@ void Call::end_call() {
       myfile.close();
     }
 
+    if (sys->get_daily_log()) {
+      std::ofstream myfile2(dailylog_filename, std::ofstream::app);
+      if (myfile2.is_open()) {
+        myfile2 << "\n" << this->get_start_time() << "," << (this->stop_time - this->get_start_time()) << "," << final_length << "," << this->talkgroup << "," << this->emergency << "," << this->priority << "," << this->duplex << "," << this->mode << ",";
+
+        for (int i = 0; i < src_list.size(); i++) {
+          myfile2 << src_list[i].source;
+          if (i < (src_list.size()-1)) {
+            myfile2 << "|";
+          }
+        }
+        myfile2 << this->get_recorder()->get_source()->get_device() << ",";
```

Comment:
What I see here is (line breaks added for readability)

```
(new line added at the start) start time, recording time, recording duration, talkgroup,
emergency, priority, duplex, mode,

...
```

and then you iterate over `src_list`, which in the base case can be of size 0 which causes nothing to be written, size 1 in which case one source is written (and nothing further), or size n, in which case `source|` is written until the last source in which case 1 is met (1 source is written and nothing further):

```
base case: (nothing is written)
1 source: source (and nothing further)
n sources: source| (n-1 times) source
```

Then you write `this->get_recorder()->get_source()->get_device()`, with no separation from the previous list of sources.

Was this your intent?

Reply:
I have `this->get_recorder()->get_source()->get_device()` commented out in my 'production' version. Is it wanted?